charming/src/series/sankey.rs

```diff
@@ -13,7 +13,7 @@ pub enum SankeyNodeAlign {
     Justify,
 }
 
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Debug)]
```

Comment:
Why would you remove the `Deserialize` trait here?

Reply:
Fair point, not really sure but I readded it and fixed all the resulting errors. This commit works on my machine [tm], so please merge it!

;-)